finalPrep/README.md

```diff
@@ -45,30 +45,36 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 - `Add Week 1 Screenshot Here`
+<img width="1200" alt="Screen Shot 2022-02-10 at 7 03 21 PM" src="https://user-images.githubusercontent.com/93043035/153527462-547f3c3d-470a-49d6-a49c-ad607f51b20c.png">
+
 - `Add Week 2 Screenshot Here`
+
+<img width="1200" alt="Screen Shot 2022-02-10 at 7 03 36 PM" src="https://user-images.githubusercontent.com/93043035/153527520-2c96657d-9100-4ce7-9bb0-2e8e6491b3b9.png">
+
 - `Add Week 3 Screenshot Here`
+<img width="1200" alt="Screen Shot 2022-02-10 at 7 03 50 PM" src="https://user-images.githubusercontent.com/93043035/153527607-0849b078-8c98-4ba1-a536-fad44a093ef4.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [See my Mentorship DTR gist here!](https://gist.github.com/ASands17/0a91cf89984d0bc95a1bcd7f6876eb6f)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [X] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [X] Add a link to your gist here: [See my Lesson Prep gist here!](https://gist.github.com/ASands17/97e0b210f6dcd624b40a87d6b46233a0)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [X] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [X] Add a link to your gist here: [See my DTR Guiding Questions gist here!](https://gist.github.com/ASands17/b049b4c7fffd1328f8de59a4ba5fd67b)
```

Comment:
This makes me feel like you are really self aware about your strengths and boundaries and set up well to communicate expectations with partners, which is HUGE!